resources/dicts/events/lifegen_events/murder.json

```diff
@@ -403,7 +411,9 @@
         "You chose v_c as deputy for their undeniable loyalty, cunning, strength, and selflessness. But over time, you noticed unsettling changes. Their attempts to mimic you, their secretive chats with warriors, all suggested a threat to your position. When v_c stormed into your den one night, your fears were confirmed. Prepared for this, you launched a preemptive attack, knowing it was either you or them."
     ],
     "murder leader elder": [
-        "Resources are tight, and as much as you didn't want it to come to this, you have no other choice. v_c is not only eating the Clan's prey, but also is an easy target. You've been laying out pieces of prey for a week now, from the yard of a Twoleg's angry dog all the way to a clearing you've set up exactly for this. You arrange to meet up with v_c at the clearing to discuss Clan matters with someone with moons of wisdom, and you sit in camp during the arranged meeting time, knowing that v_c likely wasn't coming back to camp."
+        "Resources are tight, and as much as you didn't want it to come to this, you have no other choice. v_c is not only eating the Clan's prey, but also is an easy target. You've been laying out pieces of prey for a week now, from the yard of a Twoleg's angry dog all the way to a clearing you've set up exactly for this. You arrange to meet up with v_c at the clearing to discuss Clan matters with someone with moons of wisdom, and you sit in camp during the arranged meeting time, knowing that v_c likely wasn't coming back to camp.",
+        "No one is even the bit suspiscious when you announce that v_c left to meet Silverpelt and the rest of your ancestors in their sleep, although v_c was fine and quite healthy the day before",
```

Comment:
"No one is even the bit suspiscious when you announce"
to
"No one is even a bit suspicious when you announce"